docs/Change Log.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: changelog
-title: Changelog
-sidebar_label: Changelog
+title: Change Log
```

Comment:
This is intentionally a single word

Reply:
Same with the filename

Reply:
Ok. Spell check doesn't like it but I'll change it back.